docs/zh/component_specs/sequencer/mysql.md

```diff
@@ -0,0 +1,10 @@
+# mysql
+
+## 配置项说明
```

Comment:
Please add this doc into the sidebar.
See https://mosn.io/layotto/#/zh/development/contributing-doc?id=_3-%e6%96%b0%e5%a2%9e%e6%96%87%e6%a1%a3%e9%9c%80%e8%a6%81%e5%81%9a%e5%93%aa%e4%ba%9b%e4%ba%8b
![image](https://user-images.githubusercontent.com/26001097/172526288-aecc0c66-fe01-40a0-a835-0c6ac348718a.png)